src/main/scala/mesosphere/marathon/MarathonSchedulerActor.scala

```diff
@@ -163,7 +163,10 @@ class MarathonSchedulerActor(
               .getOrElse(Future.successful(()))
           } yield ()
         }
-        else promise.future
+        else promise.future andThen {
+          case _ =>
+            self ! ScaleApp(appId, force = true)
```

Comment:
It makes sense to do this here.  However, I'm guessing it was exposed because we recently increased the default task reconciliation interval.  Created issue #1177 for decoupling the frequency of global scale operations from the task reconciliation frequency.